glide-core/src/client/value_conversion.rs

```diff
@@ -178,6 +179,42 @@ pub(crate) fn convert_to_expected_type(
             _ => Err((
                 ErrorKind::TypeError,
                 "Response couldn't be converted to an array of doubles",
+            format!("(response was {:?})", value),
```

Comment:
nit: indentation seems a bit off here

Reply:
I trusted clippy/fmt to fix that...

Reply:
`clippy` doesn't care about formatting. It gives suggestions on how to write more idiomatic and performant code. `fmt` would be responsible for styling, but it's generally pretty lax compared to other formatters (like Spotless) about things like indentation because some people really hate that (https://news.ycombinator.com/item?id=10448667).

Reply:
TIL
Thanks for describing things